Packs/AggregatedScripts/Scripts/QuarantineFile/QuarantineFile.yml

```diff
@@ -0,0 +1,65 @@
+commonfields:
+  id: quarantine-file
+  version: -1
+args:
+- name: endpoint_ids
+  description: 'List of endpoint IDs.'
+  required: true
+  isArray: true
+  type: List
+- name: file_hash
+  description: 'Hash of the file. Supported types are: SHA1, SHA256.'
+  required: true
+  type: String
+- name: file_path
+  description: 'Path of the file to quarantine.'
+  required: true
+  type: String
+- name: timeout
+  description: 'Polling timeout in seconds for the quarantine commands.'
+  required: false
+  type: Integer
+- name: quarantine_brands
+  description: |-
+    Integrations brands to use for running quarantine-file. If not provided, the command will run the command for all relevant integrations by file_hash that implement quarantine-file command.
+    For multi-select, provide a comma-separated list of integration IDs. For example: "Microsoft Defender Advanced Threat Protection, Cortex Core - IR".
+  isArray: true
+- name: verbose
+  description: Whether to retrieve a human-readable entry for every command. When set to false, human-readable will only summarize the final result.
+  auto: PREDEFINED
+  predefined:
+  - 'true'
+  - 'false'
+  defaultValue: 'false'
+comment: This script run quarantine-file command on a specific file by the agent for a file that was identified as suspicious.
+name: quarantine-file
+outputs:
+- contextPath: QuarantineFile.FilePath.
+  description: Path of the file that was quarantined.
+  type: String
+- contextPath: QuarantineFile.FileHash.
+  description: Hash of the file.
+  type: String
+- contextPath: QuarantineFile.Status
+  description: whether the command was running succussefully or not.
+  type: Boolean
+- contextPath: QuarantineFile.Result
```

Comment:
@BarGali why is Result and Status both needed in the context output?
I dont see a use of Result